subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md

```diff
@@ -0,0 +1,23 @@
+#Guidelines for checkout discussion sessions
+
+This document outlines some things that can be discussed during the checkout discussion sessions. The sessions are meant to be student driven, but some groups may be quieter than others. This is by no means an exhaustive list, other subjects are bound to come up. 
+
+## Beforehand:
+* Discussion lead creates etherpad. Post link to etherpad in the session list etherpad
```

Comment:
I was planning to have everyone use the sign-up Etherpad so that comments from all groups would be visible to all others.